test/hydration/samples/binding-input/_config.js

```diff
@@ -3,6 +3,8 @@ export default {
 		name: 'world'
 	},
 
+	skip: true,
```

Comment:
Are these tests being skipped?

Reply:
A couple of tests are skipped - the tests we're written as if their test harness was async but it wasn't and when I wanted to write a test that actually had to be async - the tests failed when I changed the test harness to be async... the change that broke the tests I skipped was to  test/hydration/index.js

Reply:
see commit  https://github.com/sveltejs/svelte/pull/4309/commits/017d0db5b25c3a16aef8cb5290ea75f6928b3e70